_products_projects/component-library.md

```diff
@@ -7,7 +7,6 @@ excerpt: Resource of common application components for developers
 image: /assets/img/projects/
 image_accessibility:
 image_icon:
-tag: Component Library
 expiration_date:
 project_url: "[]()"
```

Comment:
possibly these can be removed too if they're not used.

Reply:
It's removed already in this PR :stuck_out_tongue:

Reply:
Github disagrees. you ave not removed line 12 in this PR.

Reply:
these:
``` project_url: "[]()" ```

Reply:
![image](https://user-images.githubusercontent.com/13788498/67994807-15184f80-fcac-11e9-8e75-1827acee5cbb.png)

Reply:
![image](https://user-images.githubusercontent.com/12266/67994835-4b55cf00-fcac-11e9-8ba4-eb530b09395c.png)
Still there!

Reply:
Not too sure why you're seeing something different on your end. This screenshot shows what I'm seeing.

Reply:
And i have put big pick arrows at that line on your screen shot... It's still there.....

:confused:

Reply:
Ah, as in you meant to say remove these:
![image](https://user-images.githubusercontent.com/13788498/67994936-c9b27100-fcac-11e9-8e98-d6f8886f9940.png)

I though you meant remove the tag.

Reply:
I'll remove them in a new pr

Reply:
nope. comment is on line 12, which has this content: ` project_url: "[]()"` and i've marked in your screenshot.

Reply:
Ok, I understand now :+1: I'm actually gonna remove the ones not in use too cos the Tools card don't need some fields.